lib/features/digital_guide_view/presentation/widgets/accessibility_information_card.dart

```diff
@@ -0,0 +1,62 @@
+import "package:flutter/material.dart";
+import "package:flutter_svg/svg.dart";
+
+import "../../../../../config/ui_config.dart";
+import "../../../../../theme/app_theme.dart";
+
+class AccessibilityInformationCard extends StatelessWidget {
+  final Color color;
+  final String icon;
+  final String text;
+
+  const AccessibilityInformationCard({
+    super.key,
+    required this.color,
+    required this.icon,
+    required this.text,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: double.infinity,
+      height: DigitalGuideConfig.paddingSmall * 2 +
+          DigitalGuideConfig.difficultiesCardIconSize,
+      decoration: BoxDecoration(
+        borderRadius:
+            BorderRadius.circular(DigitalGuideConfig.borderRadiusMedium),
+        color: color,
+      ),
+      child: Row(
+        children: [
+          Padding(
+            padding: DigitalGuideConfig.symetricalPaddingSmall,
+            child: Container(
+              width: DigitalGuideConfig.difficultiesCardIconSize,
+              height: DigitalGuideConfig.difficultiesCardIconSize,
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(
+                  DigitalGuideConfig.borderRadiusSmall,
+                ),
+                color: Colors.white,
+              ),
+              child: Padding(
+                padding: DigitalGuideConfig.symetricalPaddingSmall,
+                child: SvgPicture.asset(icon),
+              ),
+            ),
+          ),
+          const SizedBox(width: DigitalGuideConfig.paddingSmall),
```

Comment:
maybe it would be good to define height separately since it's not the padding :-)